Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla32206.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 32206, "ContextActions cause memory leak: Page is never destroyed", PlatformAffected.iOS)]
+	public class Bugzilla32206 : TestNavigationPage
+	{
+		protected override void Init()
+		{
+			PushAsync(new ContentPage
+			{
+				Content = new StackLayout
+				{
+					Orientation = StackOrientation.Vertical,
+					HorizontalOptions = LayoutOptions.Center,
+					VerticalOptions = LayoutOptions.Center,
+					Spacing = 15,
+					Children =
+					{
+						new Label
+						{
+							Text = "Click Push to show a ListView. You should see a ListView page counter in the console. "
+							+ "When you hit the Back button and click Push again, GC will force garbage collection and you should see a decremented counter in the console."
+						},
+						new Button
+						{
+							Text = "Push",
+							Command = new Command(o => Navigation.PushAsync(new CustomPageY())),
+						},
+					}
+				}
+			});
+		}
+	}
+
```

Comment:
You'll need to add the `[Preserve(AllMembers = true)]` attribute to these supplemental classes in your test cases; otherwise, the [linker](https://developer.xamarin.com/guides/ios/advanced_topics/linker/#Preserving_Code) will remove them and the automated UI tests will fail.